libs/state/src/lib/core/utils/typing.ts

```diff
@@ -19,7 +19,7 @@ export function isPromiseGuard<T>(value: unknown): value is Promise<T> {
 export function isOperateFnArrayGuard<T, R = T>(
   op: any[]
 ): op is OperatorFunction<T, R>[] {
-  return op.length > 0 && op.every((i: any) => typeof i !== 'string');
+  return op.length > 0 && op.every((i: any) => typeof i === 'function');
```

Comment:
is typeof 'function' correct? isn't this a bit too generic?

Reply:
An operator is a function.

We would have to put an observable in and see if one comes out to be really sure it is an operator.
😅😅

Reply:
bad idea :)